src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2201,6 +2202,11 @@ public function getBinaryStream($path)
             array($nodeId, $propertyName, $this->workspaceName)
         );
 
+        if (count($data) === 0) {
+            throw new RepositoryException('No binary data found in stream');
```

Comment:
I'm not sure, but if it's multivalue shouldn't we get $data = [[],[]] instead of [] when theres simply no data at all?